open-machine-learning-jupyter-book/deep-learning/rnn.ipynb

```diff
@@ -159,9 +159,21 @@
     "## Code"
```

Comment:
<p>This code block is too long and needs to be split.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>